site/content/en/docs/Advanced/allocator-service.md

```diff
@@ -101,6 +101,10 @@ echo $TLS_CA_VALUE | base64 -d > ca.crt
 # echo $TLS_CA_VALUE | base64 -D > ca.crt
 ```
 
+### Bring Your Own Certificates (advanced)
```

Comment:
This won't work until you use the helm chart from the upcoming release, so it should be "hidden" in a release guard: https://agones.dev/site/docs/contribute/#within-a-page

Reply:
Neat. Fixed

Reply:
Can/should I do the same with the helm values table in the docs?

Reply:
Yes. With tables you have to copy/paste the whole table, which is super annoying. Or maybe that is what the **New Configuration Features:** table is for? @markmandel ?

Reply:
That's why it's there! So you don't have to copy paste the whole thing 😄  because it is super annoying.

Reply:
Looks like that table hasn't been merged since 1.15. Do you want me to combine that table into the main one and create a new one for 1.16?

Reply:
That would be great! I wonder if our release checklist is missing a step...

Reply:
Please change the publish version for the New Configuration Features to 1.16.0 and move the rows that were already in that table up to the visible table. Thanks!

Reply:
Done